test/context/toJSON.js

```diff
@@ -1,7 +1,7 @@
 
 'use strict';
 
-const context = require('../context');
+const context = require('.');
```

Comment:
`require('.')` might be a special cast for `process.cwd()`... not sure if that's still the case